### ISSUE TITLE
Feature: scrollSpyOnce

### ISSUE THREAD

**Issue by sascha-bleech** [inorganik/countUp.js#278]
[ x] Feature request

CountUp.js version: Latest

## Description

It would be nice if you could tell scrollSpy to only run once. Maybe scrollSpyOnce: true

**Comment by simonquest** [comment]
Yep, I support that: it'd be a really nice to have

**Comment by weavermedia** [comment]
I added this behavior by just commenting out the call to `reset` in the `handleScroll` function:
```
else if ( window.scrollY > bottomOfEl && !self.paused) {
  // scrolled past
  // self.reset(); <= comment out the reset and it will only trigger once per page load
}
```

Could add a `scrollSpyOnce` option, default to false to backwards-compatibility, then something like:
```
else if ( window.scrollY > bottomOfEl && !self.paused) {
  // scrolled past
  if (self.options.scrollSpyOnce == false) {
    self.reset();
  }
}
```

**Comment by inorganik** [comment]
Closed thanks to @adicacan's PR #282 

This is available in v2.2.0
